getting-started/services.md

```diff
@@ -19,6 +19,13 @@ In the Timescale console, you create a service to house your Timescale
 database. Each service contains a single database. If you need more
 databases, you can create additional services for each.
 
+When creating a service, you have two options: a time-series database and 
+Dynamic PostgreSQL. The time-series database is optimized for your production 
+time-series and analytics workloads. Dynamic PostgreSQL is built for all of 
+your other production database workloads. Both are 100% PostgreSQL and use 
```

Comment:
We should probably align this text with the other section https://github.com/timescale/docs/pull/2791/files#diff-404211ee40f487cbfdeb6a296be73f1daafa324b10808623c165cb6d87447a9eR28